buildSrc/src/main/groovy/com/uber/okbuck/rule/android/AndroidTestRule.groovy

```diff
@@ -40,6 +40,7 @@ final class AndroidTestRule extends AndroidRule {
                 aptDeps,
                 providedDeps,
                 aidlRuleNames,
+                null,
```

Comment:
I certainly can, but I simply have not yet. We aren't using the extensions in any of our kotlin tests so I haven't had a chance to write a test and use it.